lib/bundler/definition.rb

```diff
@@ -307,6 +325,11 @@ def to_lock
         out << "  #{p}\n"
       end
 
+      if locked_ruby_version
```

Comment:
I would think this section belongs at the bottom, right above the bundler version section?